src/build/arg/mod.rs

```diff
@@ -913,8 +926,8 @@ impl<'help> Arg<'help> {
     /// assert!(res.is_err());
     /// assert_eq!(res.unwrap_err().kind, ErrorKind::ArgumentConflict);
     /// ```
-    pub fn conflicts_with_everything(mut self) -> Self {
-        self.exclusive = true;
+    pub fn exclusive(mut self, exclusive: bool) -> Self {
```

Comment:
Why do we have `exclusive` as an argument here?

Reply:
Because all "yes or no" methods take bool as arg, see `required`, `global`, etc...